### PR TITLE
Submit button made green

### LINKED ISSUE #1251: Make application Submit button be green
We use white buttons for simple buttons and blue buttons for higher-visual-priority buttons. The Submit button is the highest priority of all, so it should get an even higher priority color, such as green.

### PR DESCRIPTION
**Pull request by soham4abc**
### Description
Submit button made green

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #1251

### CODE REVIEW

**Comment by kendrickt** [comment]
(blocking) a screenshot would be handy in the PR description.

**Comment by soham4abc** [comment]
> (blocking)
> 
> 1. a screenshot would be handy in the PR description.
> 2. We have a `views.style` package with classes like `AdminStyles` and `ApplicantStyles`. Styles should be configured there.
> 3. This change makes the button to create a new Trusted Intermediary green, which may not be what we want. The ticket is asking for making the application submit button green, which can be found in [`ApplicantProgramSummaryView`](https://github.com/seattle-uat/civiform/blob/aa0872c391193a759d61059ddc450c4afb40e293/universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java#L81) and uses [`ApplicantStyles#BUTTON_SUBMIT_APPLICATION`](https://github.com/seattle-uat/civiform/blob/aa0872c391193a759d61059ddc450c4afb40e293/universal-application-tool-0.0.1/app/views/style/ApplicantStyles.java#L108).

I got it but I am having issue while running the local server tying to run the `bin/run-dev` with docker but it's not working. Once done I will be able to give the Screenshot of my work.

**Comment by soham4abc** [comment]
Hope this is what the issue requested.

File: universal-application-tool-0.0.1/app/views/style/ApplicantStyles.java [hunk 1]
```diff
@@ -106,7 +106,7 @@
   public static final String BUTTON_REVIEW =
       StyleUtils.joinStyles(BUTTON_BASE_OUTLINE_UPPERCASE, Styles.TEXT_BASE);
   public static final String BUTTON_SUBMIT_APPLICATION =
-      StyleUtils.joinStyles(BUTTON_BASE_SOLID_UPPERCASE, Styles.TEXT_BASE, Styles.MX_AUTO);
+      StyleUtils.joinStyles(BUTTON_BASE_SOLID_UPPERCASE, Styles.TEXT_BASE, Styles.MX_AUTO,Styles.BG_GREEN_50);
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/views/style/ApplicantStyles.java:109]: (blocking) this is close, but BG_GREEN_50 is way too light to see. A screenshot of the page would be really helpful.

**Review by kendrickt — CHANGES_REQUESTED** [review]

**Comment by soham4abc** [comment]
I am trying to provide you a screenshot. But, whenever I run `bin/run-dev` even after successful compilation my `localhost:9000` remains unresponsive. Pls help me in this regard

**Comment by soham4abc** [comment]
Used bg-green-400 for better effect

**Review by kendrickt — CHANGES_REQUESTED** [review]
(blocking) it would really help to take a screenshot so you can see what this change looks like.

It would look something like this:
image

The text and background does not have enough contrast, which makes the button hard to read.

Please choose an darker green, or keep the lighter green and make the text darker.

**Comment by soham4abc** [comment]
> (blocking) it would really help to take a screenshot so you can see what this change looks like.
> 
> It would look something like this:
> image
> 
> The text and background does not have enough contrast, which makes the button hard to read.
> 
> Please choose an darker green, or keep the lighter green and make the text darker.

image

Used this darker green for better contrast hope it helps!

**Review by kendrickt — APPROVED** [review]
Looks good to me but you'll need to run `bin/fmt` to merge it in. Ping me in our slack space if you need some help with that.

**Comment by soham4abc** [comment]
> Looks good to me but you'll need to run `bin/fmt` to merge it in. Ping me in our slack space if you need some help with that.
> 
> You'll see in the `Format (java) / formatting (pull_request)` is failing for this PR.

Done, Pls have a look!

**Pull request merged**